File: PRAK205-2510817320004-RahmaSyaritaJaya.c
#include <stdio.h>
#include <math.h>

int main(){
int b, c, a;
scanf("%d %d", &b, &c);
a = sqrt(c*c - b*b);
printf("Alas = %d cm\nTinggi = %d cm\nKeliling = %d cm\nLuas = %d cm^2\n", a, b, a+b+c, (a*b)/2);

scanf("%d", &b);
scanf("%d", &c);
a = sqrt(c*c - b*b);
printf("Alas = %d cm\nTinggi = %d cm\nKeliling = %d cm\nLuas = %d cm^2\n", a, b, a+b+c, (a*b)/2);
return 0;
}

File: PRAK205-2510817320004-RahmaSyaritaJaya.py
import math

b, c = map(int, input().split())
a = math.sqrt(c*c - b*b)
print(f"Alas = {a:.0f} cm\nTinggi = {b:.0f} cm\nKeliling = {a+b+c:.0f} cm\nLuas = {0.5*a*b:.0f} cm^2")

b = float(input())
c = float(input())
print(f"Alas = {a:.0f} cm\nTinggi = {b:.0f} cm\nKeliling = {a+b+c:.0f} cm\nLuas = {0.5*a*b:.0f} cm^2")
